hikari/impl/rest.py

```diff
@@ -2256,7 +2257,7 @@ async def fetch_available_sticker_packs(self) -> typing.Sequence[stickers.Sticke
         route = routes.GET_STICKER_PACKS.compile()
         response = await self._request(route, no_auth=True)
         assert isinstance(response, dict)
-        return [self._entity_factory.deserialize_sticker_pack(pack) for pack in response["sticker_packs"]]
+        return data_binding.cast_json_array(response["sticker_packs"], self._entity_factory.deserialize_sticker_pack)
```

Comment:
isn't this just a tad bit unnecessary, kind of seems like this helper function introduces more lines of code than it saves

Reply:
Its for consistency with other functions that do this sort of this. Would it be best to remove `data_binding.cast_json_array` and just use a list cast?

Reply:
Removed :+1: